### PR TITLE
add code coverage for broker

### PR DESCRIPTION
**Pull request by xiaolanz**
```release-note
none
```

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -144,6 +144,60 @@ presubmits:
...
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: broker-codecov-token
+          mountPath: /etc/codecov
+          readOnly: true
+        - name: e2e-testing-kubeconfig
```
Comment (sebastienvas) [inline prow/config.yaml:172]: Do your test need to run on a cluster ? If not please remove the e2e-testing-kubeconfig volume and volumeMount

**Review by sebastienvas — COMMENTED** [review]
PTAL

File: prow/config.yaml (hunk 1 above)
Comment (xiaolanz) [reply prow/config.yaml:172]: yes. It runs on cluster for crd configs.

**Pull request merged**